ibis/tests/expr/test_table.py

```diff
@@ -1881,3 +1881,43 @@ def test_invalid_distinct_empty_key():
     t = ibis.table(dict(a="int", b="string"), name="t")
     with pytest.raises(com.IbisInputError):
         t.distinct(on="c", keep="first")
+
+
+def test_tumble_tvf(schema, table):
```

Comment:
`test_table` is long enough as is. I think since this functionality is new and more streaming specific, a new test file might be nice. `test_window_by.py` or something (depending on how we name the operations/types files).